samples/tutorial/Tutorial2.md

```diff
@@ -392,8 +397,9 @@ object RootWorkflow : StatefulWorkflow<Unit, State, Nothing, Any>() {
 
       // When the state is Todo, defer to the TodoListWorkflow.
       is Todo -> {
-        val todoScreen = context.renderChild(TodoListWorkflow, props = ListProps(state.username)) {
```

Comment:
ListProps is not defined yet. At this point `TodoListWorkflow` still outputs `Nothing`